### PR TITLE
 add de.gunibert.Hackgregator

### PR DESCRIPTION
**Pull request by gwutz**

### CODE REVIEW

File: de.gunibert.Hackgregator.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
+{
+    "app-id": "de.gunibert.Hackgregator",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "master",
```
Comment (bilelmoussaoui) [inline de.gunibert.Hackgregator.json]:
```suggestion
    "runtime-version": "3.34",
```

File: de.gunibert.Hackgregator.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        "--share=network",
+        "--device=dri"
+    ],
+    "build-options" : {
+        "cflags": "-O2 -g",
+        "cxxflags": "-O2 -g",
+        "env": {
+            "V": "1"
+        }
+    },
```
Comment (bilelmoussaoui) [inline de.gunibert.Hackgregator.json]: Already set by default

File: de.gunibert.Hackgregator.json [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+            ]
+        },
+        {
+            "name": "hackernews-glib",
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://gitlab.com/gunibert/hackernews-glib.git",
+                    "tag" : "0.2.0"
```
Comment (bilelmoussaoui) [inline de.gunibert.Hackgregator.json]: A commit is needed here & for a hackgreator too.

**Comment by bilelmoussaoui** [comment]
bot, build  de.gunibert.Hackgregator

File: de.gunibert.Hackgregator.json (hunk 3 above)
Comment (gwutz) [reply de.gunibert.Hackgregator.json]: I think this has already a tag. Or what do you mean by that?

**Comment by gwutz** [comment]
bot, build de.gunibert.Hackgregator

**Pull request closed without merging**